app/controllers/concerns/metrics_queries.rb

```diff
@@ -0,0 +1,48 @@
+require "active_support/concern"
+
+module MetricsQueries
```

Comment:
new concern where i've moved some reusable queries. wanted to keep these separate from the other `MetricsUtils` just to keep things somewhat organized.